melanomadetect.py

```diff
@@ -0,0 +1,15 @@
+from get_prediction import get_prediction
+
+
+
+
+class Meldetect
+    """
```

Comment:
Make sure to flesh out the comments before checking the code in. Comments will really help you outline what you need to do when you come back to it.